content/en/docs/setup/install/webhook/index.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Istio Webhook Management [Experimental]
+description: Shows how to manage webhooks in Istio through istioctl.
+weight: 100
+keywords: [security,webhook]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio has two webhooks: Galley and the sidecar injector. By default,
+these webhooks manage their own configurations. From a
+security perspective, this default behavior is not recommended because a compromised webhook could then conduct
+privilege escalation attacks.
+
+This task shows how to use `istioctl`, instead of the webhooks, to
+manage their configurations.
+
+## Before you begin
+
+* Create a Kubernetes cluster with Istio installed. In the installation,
+[`global.operatorManageWebhooks`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) should
+be set to `true`, and [DNS certificates should be configured](/docs/tasks/security/dns-cert).
+Istio installation guides can be found [here](/docs/setup/install).
+
+* Install [`jq`](https://stedolan.github.io/jq/) for JSON parsing.
+
+## Check webhook certificates
+
+To display the DNS names in the webhook certificates of Galley and the sidecar injector, run the following commands:
```

Comment:
Do one first and then the other. Ask readers display the DNS names for the Galley webhook then check the output. Then, ask readers to do the same for the sidecar injector. Remember to number all four steps and split the commands.
For readers, both the text and the commands appear way more complicated than they are.
Avoid using `|` in commands for that same reason.
Be accurate in your description of what the commands do. Not every reader will understand or be as familiar as you are with the commands and tools you are using.
```suggestion
To display the DNS names in the webhook certificates of Galley and the sidecar injector, we need to get the secret from Kubernetes, parse it, decode it, and apply the key to see the text output with the following  commands:
```

Reply:
Revised accordingly.